### ISSUE TITLE
sixt4 - Senior .NET Developer - Toronto

### ISSUE THREAD

**Issue by lowsound42** [TechnologyMasters/jobs#33]
## What You'll get

We are a well capitalized tech startup based in downtown Toronto, working on solutions to complex problems in the Travel industry. We are passionate about software development and we are continuously seeking the sharpest minds to help grow our team. Check us out at www.sixt4.io

Our TripBuilder API provides access to the most comprehensive inventory of traditional & low cost carriers allowing us to create billions of flight itineraries not available to other Travel Companies. 

### Salary Expectation

- Commensurate with experience 
- Full Time

### Benefits

- Competitive Salary and benefits package
- Free snacks, and drinks every day and a weekly catered lunch
- Flexible hours
- Downtown location with easy access to transit
- Pick your hardware (We’re both Mac and Windows friendly)

### Location

- Toronto, Ontario

## What You'll Do _(Job Description)_

- Work within a cross-functional team to design and develop solutions to complex problem
- Create technical roadmaps for the product and services that your team owns
- Mentor developers through design sessions and code reviews
- Explore new technology options to improve our applications and development processes

## What You Need to Be Successful _(Skills)_

- 2+ years development experience with .NET technologies
- Master of C#
- AWS experience including EC2, SQS, SNS, Lambda, API Gateway a plus
- Hands on experience with microservices
- N-tiered system design
- Strong background in parallel processing, concurrency and thread management
- Familiarity with database technologies such as SQL Server, Postgres, NoSQL, etc.
- Strong problem solving skills
- Startup experience a plus
- Team player with strong interpersonal and communication skills
- C++ and Linux experience are nice to haves

## How to apply

Please send an email to [Omar Khan](omar.khan@sixt4.io) with your resume, use 'Senior .NET Developer' as subject

---

## Meta

- [x] Full Time
- [ ] Part Time
- [ ] Contract
- [ ] Remote OK
- [ ] Fully Remote
- [ ] Hourly
- [x] Salary
